Source_Code/TM_Backend/TeamMentor.Rest/Controllers/TM_REST_Users.cs

```diff
@@ -257,8 +262,22 @@ public bool				RBAC_IsAdmin()
             }
             return(String.IsNullOrEmpty(errorMessage) ? "Success" : errorMessage);
         }
+        private static bool IsValidEmailAddress(string emailAddress)
+        {
+            bool isValid = false;
+            try
+            {
+                var address = new System.Net.Mail.MailAddress(emailAddress);
+                isValid = true;

```

Comment:
you don't need the **isValid** and **address** variable

just return true here